index.js

```diff
@@ -50,10 +52,21 @@ function test(p, opts) {
 }
 
 module.exports = function (patterns, options) {
+  mochaInterface = getMochaInterface();
   var contextName = (options && options.contextName) || 'eslint';
-  describe(contextName, function () {
+  mochaInterface.describe(contextName, function () {
     globAll.sync(patterns).forEach(function (file) {
       test(file, options);
     });
   });
 };
+
+function getMochaInterface () {
+  if (global.describe) {
+    return { describe: global.describe, it: global.it };
+  }
+  if (global.suite) {
+    return { describe: global.suite, it: global.test };
+  }
+  throw new Error('no mocha interface was found on the global scope');
```

Comment:
Will users need to change the way they run mocha-eslint with this change?  I personally no longer use Mocha, so I'm a bit out-of-the-loop on this change.  Basically, does the readme need to change in this package, do you think?

Reply:
Hi @IanVS. No that should not be necessary, as you should still use mocha-eslint as you have always done. 
In the current version of mocha-eslint, if you run the test and mocha is not available then an error is thrown when it tries to use describe. So the only change is that the error you see here in the change is thrown instead.